rust/cubestore/cubestore/src/metastore/mod.rs

```diff
@@ -137,16 +149,28 @@ macro_rules! base_rocks_secondary_index {
                 RocksSecondaryIndex::version(self)
             }
 
+            fn value_version(&self) -> u32 {
+                RocksSecondaryIndex::value_version(self)
+            }
+
             fn is_unique(&self) -> bool {
                 RocksSecondaryIndex::is_unique(self)
             }
+
+            fn is_ttl(&self) -> bool {
+                RocksSecondaryIndex::is_ttl(self)
+            }
+
+            fn get_expire<'a>(&self, row: &'a $table) -> &'a Option<chrono::DateTime<chrono::Utc>> {
+                RocksSecondaryIndex::get_expire(self, row)
+            }
         }
     };
 }
 
 #[macro_export]
 macro_rules! rocks_table_impl {
-    ($table: ty, $rocks_table: ident, $table_id: expr, $indexes: block) => {
+    ($table: ty, $rocks_table: ident, $table_id: expr, $indexes: block, $cfn: expr) => {
```

Comment:
May be `$cfn` -> `$column_family` ?  It's not obvious what `cfn` means then you're looking at the signature

Reply:
Sure! Renamed.